pkg/bot/slack.go

```diff
@@ -377,3 +369,27 @@ func (b *Slack) findAndTrimBotMention(msg string) (string, bool) {
 func mdHeaderFormatter(msg string) string {
 	return fmt.Sprintf("*%s*", msg)
 }
+
+func sendMessageWithFileUpload(channel string, resp interactive.Message, client *slack.Client) error {
+	uploadMsgOpts := []slack.MsgOption{
+		slack.MsgOptionText(resp.Description, false),
+		slack.MsgOptionAsUser(true),
+	}
+	if _, _, err := client.PostMessage(channel, uploadMsgOpts...); err != nil {
+		return fmt.Errorf("while posting attachment message: %w", err)
+	}
+
+	params := slack.FileUploadParameters{
+		Filename: "Response.txt",
+		Title:    "Response.txt",
+		Content:  interactive.MessageToPlaintext(resp, interactive.NewlineFormatter),
+		Channels: []string{channel},
+	}
```

Comment:
Interesting, I didn't realise `InitialComment: resp.Description,` was an option.

Reply:
From https://github.com/kubeshop/botkube/issues/750 ... when I read:

> When a message is sent as file, we should render it as plaintext. The header should be removed from the file and printed directly in the message.

I thought that meant using 2 messages.

Reply:
>The header should be removed from the file and printed directly in the message. 

Sorry, I cannot imagine how it can be interpreted in this way 🤔  What's more, what would be the purpose of it? 

Using 2 messages is so crazy for such thing.